docker/entrypoint_check.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh -l
+
+echo '\nGetting the code...\n'
+git clone --quiet https://$INPUT_TOKEN@github.com/$GITHUB_REPOSITORY check
+cd check
+git checkout $BRANCH_SOURCE
+rm .Rprofile
+
+echo '\nSession info\n'
+Rscript -e 'sessioninfo::session_info()'
+
+echo '\nUpdate version number...\n'
+git config --global user.email "info@inbo.be"
+git config --global user.name "INBO"
+UPDATED=$(Rscript -e 'protocolhelper::update_version_number("'$PROTOCOL_CODE'")')
+echo 'output updated:' $UPDATED
+if [ "$UPDATED" = "[1] TRUE" ]; then
+  # remake the last commit without deletion of .Rprofile
+
+  git push -f
+  echo '\ncommit with new version pushed\n'
+fi
+
+echo '\nChecking protocols specific tests...\n'
+
+Rscript "docker/check_all.R"
```

Comment:
the script is missing from the Dockerimage

Reply:
The docker loads it from repo protocolsource, so it can be adapted without rebuilding the dockerimage.  (Error of @ThierryO when running `update_website` occurred because this file is not yet added in the main branch.)